docs/build_docs.sh

```diff
@@ -193,7 +193,9 @@ if [[ "$pdf_required" == "True" ]]; then
     which_latex=`which latex`
     if [[ "$which_latex" == "" ]]; then
         echo "ERROR: latex must be installed in order to create pdf documentation."
-        echo "  try 'module load latex'"
+        echo "  try 'conda install latexcodec' if in anaconda"
```

Comment:
@biosafetylvl5  Note I updated this from module load latex to conda install latexcode - that is the actual conda command required to install latex.

Reply:
Thanks! What was `module load latex` from?

Reply:
`module load` is a command used on some linux systems to "load" a plugin by changing environment variables like Paths.